tests/commons/j2/test_j2_functions.py

```
@@ -272,6 +275,101 @@
     assert fmt == exp, "unexpected uri format for nobase"
 
 
+@pytest.mark.parametrize(
+    "case",
+    [
+        (
+            (
+                datetime(2024, 1, 13, 11, 48, 29),
```

Ruff (0.8.2)

`datetime.datetime()` called without a `tzinfo` argument

(DTZ001)